### ISSUE TITLE
Unfair Comparison in your paper 

### ISSUE THREAD

**Issue by LarryBrid** [nv-tlabs/GSCNN#2]
After reading your paper and discussion, we have few questions. 
1. In the leaderboard, we can see that you use Mapillary data. However, In the paper of table 6. You compared TKCN and AAF-PSP which didn't use Mapillay data. **It is obviously unfair.** 
So what is real result without Maypiilary data and only use fine-data on Cityscapes test set ?
2. Also we didn't  find your claim in using Mapillary data in your paper why didn't mention it?
3. In the table 3, Why mIoU is so low when use your ResNet101 backbone according other repo. ResNet101 based deeplabv3+ should be 78+. Repo: https://github.com/speedinghzl/pytorch-segmentation-toolbox

**Comment by tovacinni** [comment]
Thanks for pointing out the missing information on Mapillary dataset in the paper. We will include it in our revision. 

Different works has different training setups and it is hard to compare with same settings as previous works. Our ablations (Table 3) show consistent relative improvements with the same training setups that aren't necessarily optimizing for maximum performance.

**Comment by XiaLiPKU** [comment]
I think you should mention the usage of Mapillay dataset in your paper, or at least in the code or the leaderboard

**Comment by tovacinni** [comment]
We do mention it on the leaderboard, and we also updated our paper to mention the usage for our camera-ready.

**Comment by Wangzhuoying0716** [comment]
@tovacinni Have you tested not using Mapillary dataset and how about the performance? Thanks a lot and will you release the official testing code?

**Comment by lxtGH** [comment]
@tovacinni Have you tested not using Mapillary dataset and how about the performance? Thanks a lot and will you release the official testing code?
